src/query.js

```diff
@@ -1950,12 +1950,28 @@ function Equals() {
  * @param {module:query~ExprArg} in
  *   An object to search against.
  * @return {Expr}
+ *
+ * @deprecated use ContainsField instead
  */
 function Contains(path, _in) {
   arity.exact(2, arguments, Contains.name)
   return new Expr({ contains: wrap(path), in: wrap(_in) })
 }
 
+/**
+ * See the [docs](https://app.fauna.com/documentation/reference/queryapi#miscellaneous-functions).
+ *
+ * @param {module:query~ExprArg} path
+ *   An array representing a path to check for the existence of.
+ * @param {module:query~ExprArg} in
+ *   An object to search against.
+ * @return {Expr}
+ */
+function ContainsField(path, _in) {
+  arity.exact(2, arguments, ContainsField.name)
+  return new Expr({ contains_field: wrap(path), in: wrap(_in) })
```

Comment:
**Note:** To get our unit tests to pass, `contains_field` must remain as `contains`

Reply:
@nathaniel-may @BrunoQuaresma I see that we've added `ContainsField` to the Core code, however, I'm not sure if it's disabled or not being served to us properly since causes our tests to fail.

I'm assuming the name should either be `contains_field` or `containsfield` 🤔

Reply:
you need to update api version to 3, otherwise the function won't be available